.bench/Strings-report-github.md

```diff
@@ -0,0 +1,16 @@
+``` ini
+
+BenchmarkDotNet=v0.10.4, OS=Windows 10.0.10586
+Processor=Intel Core i7-3770 CPU 3.40GHz (Ivy Bridge), ProcessorCount=8
```

Comment:
you need better machine

Reply:
I’ve been waiting for over a year to reply to this - and now that I’ve bought some Raspberry PIs, I can finally say that I do indeed have some better hardware lol

PS Hotfix now runs on a Raspberry Pi cluster under docker using dotnet core ;)

Reply:
PS I’m gonna drop the mike and resolve this conversation :p